### DISCUSSION TITLE
master image selection

### DISCUSSION THREAD

**Discussion by sincaliz** [gmtsar/gmtsar#950] [General]
Hi. I would like to ask if the master image selected for pre_proc.csh (creation of auxiliary files from raw data) must be the same used for following steps using batch_processing.csh SAT master list_images step. I am trying to run step by step to understand the workflow. Are there some errors if  the master image on pre_proc.csh and batch_processing.csh differ?

**Comment by Xiaohua-Eric-Xu** [comment]
Usually you need to use the same master image especially for the early dataset such as ERS/ENVISAT/ALOS, as they need doppler info from the master image.
